src/Main.java

```diff
@@ -0,0 +1,62 @@
+import task.BusinessTrip;
+
+
+//this is the code from stage 2. It's the same as on stage 1.
+//Since all the requirements from stage 2-file were already met - i didn't change anything.
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        BusinessTrip[] BusinessTrips = new BusinessTrip[]{
+                new BusinessTrip("Sergey Rachmaninoff", 2200, 15),
+                new BusinessTrip(null, 0, 1),
```

Comment:
```suggestion
                null,
```
это пустой элемент. 
по условию индекс 2 у него в массиве.